renv/activate.R

```diff
@@ -0,0 +1,349 @@
+
```

Comment:
Should the renv folder be part of the project? I'm only minimally familiar with this, but thought it had more to do with local copies of a project and wasn't relevant to broader distribution of a package.